web/src/main/java/org/mskcc/oncotree/api/TumorTypesTxtApi.java

```diff
@@ -32,8 +36,13 @@
         produces = {"text/plain"},
         method = RequestMethod.GET)
     public ResponseEntity<InputStreamResource> tumorTypesTxtGet(
+        @ApiParam(value = "The version of tumor types. For example, 1, 1.1 Please see GitHub for released versions. ")
```

Comment:
This example value is not correct ... let's use oncotree_latest_stable ... or better yet, call VersionUtil.getDefaultVersion() to give an example.